### PR TITLE
feat: add tip for arbitrary Feedback metadata

### PR DESCRIPTION
**Pull request by elliotfontaine**
> **Tip**
> If you'd like to attach additional metadata to the feedback (player props, game state, etc.), you could always append `feedback_comment` with said metadata, before giving it to the `send()` function.

Sorry about the merge commit. I'd suggest squashing the two.

### CODE REVIEW

File: docs/godot/feedback.md [hunk 1]
```diff
@@ -40,3 +40,7 @@ func _on_pressed() -> void:
 	await Talo.feedback.send(internal_name, feedback_comment)
 	print("Feedback sent for %s: %s" % [internal_name, feedback_comment])
 ```
+
+:::tip
+If you'd like to attach additional metadata to the feedback (player props, game state, etc.), you could always append `feedback_comment` with said metadata, before giving it to the `send()` function.
```
Comment (tudddorrr) [inline docs/godot/feedback.md:45]: would just tweak the wording slightly here to make it read a little better:
"You can attach additional metadata  (player props, game state, etc.) to feedback by appending it to the `feedback_comment` parameter before calling `send()`."

**Review by tudddorrr — CHANGES_REQUESTED** [review]

File: docs/godot/feedback.md (hunk 1 above)
Comment (elliotfontaine) [reply docs/godot/feedback.md:45]: Yeah you're right, the wording wasn't good. What about :
> [...] to the `feedback_comment` String before calling `send()`.

Would emphasize the fact that you're appending to a String, and you'll probably want to typecast your metadata to String beforehand.

Comment (tudddorrr) [reply docs/godot/feedback.md:45]: yeah that works!

Comment (tudddorrr) [reply docs/godot/feedback.md:45]: @elliotfontaine any chance you can update this soon?

Comment (elliotfontaine) [reply docs/godot/feedback.md]: Whoops sorry, completely missed the ping  :/

Comment (tudddorrr) [reply docs/godot/feedback.md]: no problem lets get this merged and I'll update the unity docs. thanks!

**Pull request merged**